script.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pprint
+import config
+
+def main():
+    token = generateToken(config.client_id, config.client_secret)
```

Comment:
In Python, we are following the [pep8 style guide](https://www.python.org/dev/peps/pep-0008/) as the coding standard. Base on it, we should use `snake_case` as the naming convention for the variables and functions instead of the `camelCase`.